CMakeLists.txt

```diff
@@ -411,6 +416,23 @@ endif()
 
 # CXX Compiler settings
 set(CMAKE_CXX_STANDARD 14)
+
+if(OPENMW_LTO_BUILD)
+    if(POLICY CMP0069)
```

Comment:
Now we both have a better idea of what's going on, I'd not check the policy here, just the version. If the version is high enough, we can still do the check, even if we don't use the result of it to decide what to do, just what to log.